charts/csi-vxflexos/values.yaml

```diff
@@ -163,7 +163,7 @@ controller:
   # volumeNamePrefix- defines a string prepended to each volume created by the CSI driver.
   # Default value: none
   # Examples: "k8s", "app1"
```

Comment:
At least add a example with the default value, same for other files and the values that we are modifying here as part of this PR.